frame/babe/src/mock.rs

```diff
@@ -195,7 +195,8 @@ impl pallet_staking::Config for Test {
 	type SessionInterface = Self;
 	type UnixTime = pallet_timestamp::Pallet<Test>;
 	type EraPayout = pallet_staking::ConvertCurve<RewardCurve>;
-	type MaxNominatorRewardedPerValidator = ConstU32<64>;
+	type MaxExposurePageSize = ConstU32<64>;
+	type MaxExposurePageCount = ConstU32<1>;
```

Comment:
Would there be any issues making these `ConstU16`? 32 bits may be overkill for these numbers.

Reply:
An update, I want to bound EraStakersPaged and ClaimedRewards with these two values and boundedVec needs to be bounded by a u32 value. I will undo the changes to make it u16.